dcos_launch/gce.py

```diff
@@ -0,0 +1,87 @@
+import os
+import logging
+import dcos_launch.util
+from dcos_test_utils import gce
```

Comment:
Here's the style guide reference: https://google.github.io/styleguide/pyguide.html#Imports

`Use imports for packages and modules only.`